core/obsolete.php

```diff
@@ -203,4 +203,6 @@
 # changes in 2.0.0dev
 config_obsolete( 'icon_path' );
 config_obsolete( 'bug_print_page_fields' );
+config_obsolete( 'calendar_js_date_format' );
```

Comment:
should be
`config_obsolete( 'calendar_js_date_format', 'datetime_picker_format' );`